core/src/test/java/org/apache/calcite/rel/rel2sql/RelToSqlConverterTest.java

```diff
@@ -7876,6 +7878,26 @@ private void checkLiteral2(String expression, String expected) {
         .withSpark().ok(sparkExpected);
   }
 
+  /**
+   * Test case for ST_SRID function.
+   * All the spatial functions where the arguments are the same type should behave similarly.
+   */
+  @Test void testPostgisStSrid() {
+    String query = "select st_srid(\"point\") FROM \"points\"";
+    String expectedPostgis = "SELECT \"ST_SRID\"(\"point\")\nFROM \"GEO\".\"points\"";
```

Comment:
Some UT tests failed

Reply:
I should probably have created a draft PR.